Shared/Samples/Show device location with NMEA data sources/FileNMEASentenceReader.swift

```diff
@@ -14,91 +14,93 @@
 
 import Foundation
 
-/// A data source simulating a hardware that emits NMEA data.
-class FileNMEASentenceReader {
-    /// The playback time interval.
-    private let interval: TimeInterval
-    
-    /// An iterator to hold and loop through the mock NMEA data.
-    private var nmeaDataIterator: CircularIterator<Data>
-    
-    /// A timer to periodically provide NMEA data updates.
-    private var timer: Timer?
-    
-    /// An asynchronous stream of NMEA data.
-    private(set) var messages: AsyncStream<Data>!
-    
-    /// A Boolean value specifying if the reader is started.
-    private(set) var isStarted = false
-    
-    /// Loads locations from NMEA sentences.
-    /// Reads mock NMEA sentences line by line and group them by the timestamp.
-    /// - Parameters:
-    ///   - nmeaSourceFile: The URL of the NMEA source file.
-    ///   - speed: The playback time interval in second.
-    init(url: URL, interval: TimeInterval = 1.0) {
-        // An empty container for NMEA data.
-        var dataBySeconds = [Data]()
+extension ShowDeviceLocationWithNMEADataSourcesView {
+    /// A data source simulating a hardware that emits NMEA data.
+    class FileNMEASentenceReader {
+        /// The playback time interval.
+        private let interval: TimeInterval
+        
+        /// An iterator to hold and loop through the mock NMEA data.
+        private var nmeaDataIterator: CircularIterator<Data>
+        
+        /// A timer to periodically provide NMEA data updates.
+        private var timer: Timer?
+        
+        /// An asynchronous stream of NMEA data.
+        private(set) var messages: AsyncStream<Data>!
+        
+        /// A Boolean value specifying if the reader is started.
+        private(set) var isStarted = false
+        
+        /// Loads locations from NMEA sentences.
+        /// Reads mock NMEA sentences line by line and group them by the timestamp.
+        /// - Parameters:
+        ///   - nmeaSourceFile: The URL of the NMEA source file.
+        ///   - speed: The playback time interval in second.
+        init(url: URL, interval: TimeInterval = 1.0) {
+            // An empty container for NMEA data.
+            var dataBySeconds = [Data]()
```

Comment:
Not a big deal, but the syntactic sugar for arrays/dictionaries isn't intended to be used when initializing a type (per Apple devs on the forums).

```suggestion
            var dataBySeconds: [Data] = []
```

Reply:
In an effort to keep the changes in this PR related to access level, I created an issue for these. That way, they can all be addressed at once. 

- #493

Reply:
Thank you!